File: RedSheet.DbEntities/Models/vProjectModuleAssignee.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vProjectModuleAssignees", Schema = "dbo")]
	[Serializable]
	public partial class vProjectModuleAssignee  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int ProjectModuleAssigneeId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int ProjectModuleId { get; set; } 
 
	[MaxLength(50)]
	public string UserName { get; set; } 

	public vProjectModuleAssignee()
	{
    }

    }
}

File: redsheet-portal/src/app/database-models/v-project-module-assignee.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vProjectModuleAssignee {
    constructor(vProjectModuleAssignee?: vProjectModuleAssignee )  {
        let properties = [ "projectModuleAssigneeId", "projectModuleId", "userName",];
        for (let property of properties)
            if (vProjectModuleAssignee && vProjectModuleAssignee[property])
                this[property] = vProjectModuleAssignee[property];
    }
 
	projectModuleAssigneeId : number =   0 ;
 
    @range(1,2147483647)
	projectModuleId : number =   undefined;
 
    @maxLength(50)
	userName : string =   undefined;


}
